tests/test_back_2.js

```diff
@@ -51,7 +51,7 @@ test('recording', {skip: process.env.AIRPLANE}, function(t) {
 });
 
 test('passes custom options to recorder', {skip: process.env.AIRPLANE}, function(t) {
-  nockBack('recording_test.json', { recorder: { enable_reqheaders_recording: true } }, function(nockDone) {
+  nockBack('recording_test.json', { recorder: { enable_reqheaders_recording: true, output_objects: true } }, function(nockDone) {
```

Comment:
Could you please add a test case asserting identical behavior when `output_objects: true` is missing? Otherwise  we are missing an assertion for the default behavior.